### PR TITLE
feat(#158): integration of user management tool into civ project

### PR DESCRIPTION
**Pull request by aloundoye**
#158: integrate the user management tool into the CIV project as discussed in https://github.com/medic/config-moh-civ/issues/163.

### CODE REVIEW

**Comment by aloundoye** [comment]
Hi @inromualdo can you check if the contacts needed are added?
@kennsippell your feedback will be appreciated

File: src/config/chis-civ/config.json
Comment (kennsippell) [inline src/config/chis-civ/config.json:51]: Should you ask for first name before last? Like how it is said?

File: src/config/chis-civ/config.json
Comment (kennsippell) [inline src/config/chis-civ/config.json:124]: can the CHW site name be generated? 
if so, then username_from_place can also be false

File: src/config/chis-civ/config.json
Comment (kennsippell) [inline src/config/chis-civ/config.json:103]: why do you deactivate for CHWs but not for supervisors? Should the strategy be consistent for your project?

File: src/config/chis-civ/config.json
Comment (aloundoye) [reply src/config/chis-civ/config.json:46]: Yes it is but it works well.

File: src/config/chis-civ/config.json
Comment (aloundoye) [reply src/config/chis-civ/config.json:51]: Done

File: src/config/chis-civ/config.json
Comment (aloundoye) [reply src/config/chis-civ/config.json]: Done

File: src/config/chis-civ/config.json
Comment (aloundoye) [reply src/config/chis-civ/config.json]: Thanks for pointing out the type: "phone" as a possibility

File: src/config/chis-civ/config.json
Comment (aloundoye) [reply src/config/chis-civ/config.json]: I missed this one

File: src/config/chis-civ/config.json
Comment (aloundoye) [reply src/config/chis-civ/config.json]: Done

**Comment by aloundoye** [comment]
Hi both,
@kennsippell sorry for the delay I implemented your feedback, thanks.
@inromualdo is it good on your side?

**Review by inromualdo — CHANGES_REQUESTED** [review]
Nice work. I left some comments

File: src/config/chis-civ/config.json
Comment (aloundoye) [reply src/config/chis-civ/config.json:96]: @inromualdo Since the user management will generate a username is it necessary to have a field for the username?

**Comment by kennsippell** [comment]
@aloundoye @inromualdo What is the status here? Are we stalled?
Are you working together for approval or should I review and approve?

**Comment by aloundoye** [comment]
Hi @kennsippell, just waiting for approval from @inromualdo who requested some changes.

**Review by inromualdo — APPROVED** [review]
@aloundoye , we're good on this.
Thank you

**Pull request merged**
